Src/java/elm-fhir/src/main/java/org/cqframework/cql/elm/requirements/fhir/utilities/constants/CqfmConstants.java

```diff
@@ -0,0 +1,27 @@
+package org.cqframework.cql.elm.requirements.fhir.utilities.constants;
+
+// constants defined in the Quality Measures IG: http://hl7.org/fhir/us/cqfmeasures
+public class CqfmConstants {
+
+    private CqfmConstants() {}
+
+    // Extensions
+    public static final String PARAMETERS_EXT_URL =
+            "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-parameter";
+    public static final String DATA_REQUIREMENT_EXT_URL =
+            "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-dataRequirement";
+    public static final String RELATED_REQUIREMENT_EXT_URL =
+            "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-relatedRequirement";
+    public static final String DIRECT_REF_CODE_EXT_URL =
+            "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-directReferenceCode";
+    public static final String LOGIC_DEFINITION_EXT_URL =
+            "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-logicDefinition";
+    public static final String PERTINENCE_EXT_URL =
+            "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-pertinence";
```

Comment:
Having a devil of a time finding this... I see the Jira issue to implement this extension, but the url leads to a 404 and I cannot find it in the IGs.

Reply:
Right, there is currently no standardized definition for this, needs to be fleshed out still, this is a proposed implementation